libs/application/templates/parental-leave/src/lib/parentalLeaveUtils.ts

```diff
@@ -61,6 +61,7 @@ import {
   minimumPeriodStartBeforeExpectedDateOfBirth,
   multipleBirthsDefaultDays,
 } from '../config'
+import { isAfter, isBefore, isEqual, subDays, subMonths } from 'date-fns'
```

Comment:
Linting issue: need to change to 'import isAfter from 'date-fns/isAfter' and so on